Content.Server/Nutrition/EntitySystems/DrinkSystem.cs

```diff
@@ -74,6 +74,38 @@ private void OnExamined(EntityUid uid, DrinkComponent component, ExaminedEvent a
             var openedText =
                 Loc.GetString(IsEmpty(uid, component) ? "drink-component-on-examine-is-empty" : "drink-component-on-examine-is-opened");
             args.Message.AddMarkup($"\n{Loc.GetString("drink-component-on-examine-details-text", ("colorName", color), ("text", openedText))}");
+            if (!IsEmpty(uid, component))
+            {
```

Comment:
Does the default ever get hit if IsEmpty is true?

Reply:
True, I figured it could never get to the default in this case. I figured it wouldn't hurt to leave a default anyway, but I can certainly remove it since logically it should never happen.